openlibrary/templates/type/author/edit.html

```diff
@@ -106,7 +106,7 @@ <h1>$_("Edit Author")</h1>
                                 <textarea name="author--alternate_names" id="alternate_names" rows="6">$"; ".join(page.alternate_names)</textarea>
                             </div>
                         </div>
-                        <div class="label">Identifiers</div>
+                        <div class="label">Identifiers <a href="https://openlibrary.org/help/faq/editing.en#author-identifiers-purpose"><img src="/images/icons/icon_help.png" alt="$_('Author Identifiers Purpose')"/></a> </div>
```

Comment:
```suggestion
                        <div class="label">Identifiers <a href="/help/faq/editing.en#author-identifiers-purpose"><img src="/images/icons/icon_help.png" alt="$_('Author Identifiers Purpose')"/></a> </div>
```
Changing this to a relative link, as we also have [testing environments](https://testing.openlibrary.org).